### PR TITLE
add 5.12.1 release notes

### LINKED ISSUE #9103: Release notes for 5.12.1
Yesterday we came to a decision that we'd like to put out a point release. One of the primary motivators is the following bug (introduced in 5.11) that unfortunately results in a poor user experience for "remote" (Shibboleth, OAuth, and OIDC) users who log into Dataverse for the first time and see something like this (OAuth via GitHub [example](https://github.com/IQSS/dataverse/issues/9029#issuecomment-1293905548)):

198368695-3bda296d-14f8-4f83-96ef-d49b176a632b

- #9029

I renamed the milestone from 5.13 to 5.12.1: https://github.com/IQSS/dataverse/milestone/106

We'll branch from "develop" rather than making a hot fix. That means:

- everything merged into develop since 5.12 will be part of the release: https://github.com/IQSS/dataverse/pulls?q=milestone%3A5.12.1+is%3Amerged
- every pull request tagged with 5.12.1 should be part of the release: https://github.com/IQSS/dataverse/pulls?q=is%3Aopen+is%3Apr+milestone%3A5.12.1
- we plan include the following open 5.12.1 issues in the release, making pull requests as necessary: https://github.com/IQSS/dataverse/issues?q=is%3Aopen+is%3Aissue+milestone%3A5.12.1

This issue is about writing up the release notes, as usual. You can preview #9024 at https://dataverse-guide--9024.org.readthedocs.build/en/9024/developers/making-releases.html#write-release-notes

### PR DESCRIPTION
**Pull request by pdurbin**
**Which issue(s) this PR closes**:

- Closes #9103

### CODE REVIEW

File: doc/release-notes/5.12.1-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Dataverse Software 5.12.1
+
+This release brings new features, enhancements, and bug fixes to the Dataverse Software. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+## Bug Fix for "Internal Server Error" When Creating a New Remote Account
+
+Unfortunately, as of 5.11 new remote users have seen "Internal Server Error" when creating an account or checking notifications. By "remote" user we mean any user that is not creating a builtin/local account. That is, remote users log in with institutional (Shibboleth), OAuth (ORCID, GitHub, or Google) or OIDC providers.
```
Comment (scolapasta) [inline doc/release-notes/5.12.1-release-notes.md:9]: A little wordy - 2nd and 3rd sentence basically same thing. Mayve Just remove the 2nd sentence and slight ly change thirst to "Remote users are those who log in with institutional (Shibboleth), OAuth (ORCID, GitHub, or Google) or OIDC providers."

**Review by scolapasta — CHANGES_REQUESTED** [review]

File: doc/release-notes/5.12.1-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+Now, the production date is used for harvested dataset in addition to distribution date when harvesting with the `oai_dc` format.
+
+### Publication Date Now Used for Harvested Dataset if Production Date is Not Set (`oai_dc` format)
+
+For exports and harvesting in `oai_dc` format, if "Production Date" is not set, "Publication Date" is now used instead. This change is reflected in the [Dataverse 4+ Metadata Crosswalk][] linked from the [Appendix][] of the User Guide.
+
+[Dataverse 4+ Metadata Crosswalk]: https://docs.google.com/spreadsheets/d/10Luzti7svVTVKTA-px27oq3RxCUM-QbiTkm8iMd5C54/edit#gid=1901625433&range=K7
+[Appendix]: https://guides.dataverse.org/en/5.12.1/user/appendix.html
+
+### Ability to Disable OAuth Sign Up While Allowing Existing Accounts to Log In
```
[the remark this answers is not in the document]
Comment (pdurbin) [reply doc/release-notes/5.12.1-release-notes.md]: Fixed in b86ac38.

File: doc/release-notes/5.12.1-release-notes.md (hunk 1 above)
Comment (pdurbin) [reply doc/release-notes/5.12.1-release-notes.md]: Yes. Good idea. Fixed in b2a9756.

**Review by scolapasta — APPROVED** [review]

**Comment by poikilotherm** [comment]
I know this might be not in scope, but how about adding a hint on the new Payara releases and that we are brewing sth. in preparation for 6?

File: doc/release-notes/5.12.1-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+```shell
+    $PAYARA/bin/asadmin deploy dataverse-5.12.1.war
+```
+
+8\. Restart payara
+
+```shell
+    service payara stop
+    service payara start
+```
```
Comment (poikilotherm) [inline doc/release-notes/5.12.1-release-notes.md:109]:
```suggestion
```

## Outlook
With the release of [Payara 6.2022.1](https://github.com/payara/Payara/releases/tag/payara-server-6.2022.1), the days of free-to-use [Payara 5.x Community versions are counted](https://blog.payara.fish/whats-new-in-the-november-2022-payara-platform-release). Moving Dataverse to the new platform will require substantial efforts from the IQSS team and community members, as this also means shifting Dataverse to be a [Jakarta EE 10](https://jakarta.ee/release/10/) application. We are currently shaking out the details and spread news as soon as we have them. Rest assured we will do our best to provide you with a smooth transition.

```

**Comment by pdurbin** [comment]
@poikilotherm good idea. Thanks for giving me some text to start with. Added in bebd6e7.

**Pull request merged**